mne/viz/epochs.py

```diff
@@ -894,19 +894,24 @@ def plot_epochs(epochs, picks=None, scalings=None, n_epochs=20, n_channels=20,
                   scalebars_visible=False,
                   window_title=title,
                   xlabel='Epoch number')
-    fig = _browse_figure(**params)
-    fig._update_picks()
 
-    # make channel selection dialog, if requested (doesn't work well in init)
-    if group_by in ('selection', 'position'):
-        fig._create_selection_fig()
+    fig = _get_browser(**params)
+
+    if get_browser_backend() == 'matplotlib':
```

Comment:
This block should wrap every method-call which is currently not implemented in the pyqtgraph-browser (that is why the selection-code is in there too because this is not implemented in pyqtgraph yet).

As you say some methods may be difficult or unnecessary to be transfered to the pyqtgraph-backend. Maybe a common and abstract `update`-method could make the generalization easier here?

Reply:
> This block should wrap every method-call which is currently not implemented in the pyqtgraph-browser

why?  This PR is supposed to be a pure conceptual refactoring of existing matplotolib plotting, and is not supposed to implement the pyqtgraph browser yet, so there should be no need.